### PR TITLE
Chart.5

### PR DESCRIPTION
**Pull request by olu144**
Add graph to display data based on user survey results about seasons. This implementation does not require an additional servlet.

### CODE REVIEW

File: portfolio/src/main/webapp/chart.html [hunk 1]
```diff
@@ -1,12 +1,25 @@
...
+    <script src="https://www.gstatic.com/charts/loader.js"></script>
+    <script src="script.js"></script>
+  </head>
+  <body onload=loadChartsApi();loadChartsApi2();>
+    <h1>Amazon Stock Price 2014-2018</h1>
+    <div id="chart-container"></div>
+    <h1>Favorite Seasons</h1>
+    <div id="chart2-container"></div>
+    <h2>Vote</h2>
+    <form action="/season-data" method="POST">
```
Comment (dborkan) [inline portfolio/src/main/webapp/chart.html:14]: Is there a `/season-data` servlet that needs to be added to this PR?  When I tried this code I got a 404 error after clicking the submit button below

**Review by dborkan — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/chart.html (hunk 1 above)
Comment (olu144) [reply portfolio/src/main/webapp/chart.html:14]: addressed in "added seasons servlet". On my end everything functions fine even in the absence of the servlet, which is odd.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -79,28 +79,49 @@ async function isLoggedIn() {
...
+      'width':1200,
+      'height':1000
+    };
+    const chart = new google.visualization.LineChart(document.getElementById('chart-container'));
+    chart.draw(data, options);
+  });
+}
+
+/** Fetches season data and uses it to create a chart. */
+function drawAnimalChart() {
```
Comment (dborkan) [inline portfolio/src/main/webapp/script.js]: I think this function name needs to be updated

**Review by dborkan — APPROVED** [review]
Small change requested to one of the function names, but once that is done this code looks good to merge 👍

**Pull request merged**